src/commands/ref_command_handler.js

```diff
@@ -17,9 +17,8 @@ module.exports = class ReferenceCommandHandler extends CommandHandler {
 
 function cppReference(message, args) {
     const channel = message.channel;
-    if (args.length < 1) {
```

Comment:
Please put these brackets back, IMO it is a lot more readable with them there.

Reply:
I'm sorry, it's probably the habit of doing it, didn't even realize.